runtime/near-vm-runner/src/near_vm_runner.rs

```diff
@@ -237,7 +237,7 @@ impl NearVmConfig {
 //  major version << 6
 //  minor version
 const VM_CONFIG: NearVmConfig = NearVmConfig {
-    seed: (2 << 10) | (0 << 6) | 0,
```

Comment:
Uhhh reading this I feel like I misunderstood the scope of this PR, so I’d be reviewing it wrong if I were to continue (more tests should not lead to compilation changes).

Stopping my review here, let me know when I should review again :)

Reply:
This PR was originally based on the PR that upgraded the toolchain.